src/test/java/converter/UserToStringConverterTest.java

```diff
@@ -0,0 +1,35 @@
+package converter;
+
+import model.User;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class UserToStringConverterTest {
+
+    private final UserToStringConverter userToStringConverter = new UserToStringConverter();
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertNull() {
+        userToStringConverter.convert(null);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertFieldsNull() {
+        User user = new User(null, null, null, null);
+
+        userToStringConverter.convert(user);
+    }
+
+    @Test
+    public void testConvert() {
+        String expected = "igor=igor_password=ihar=osipov";
+        User user = new User("igor", "igor_password", "ihar", "osipov");
+
+        String actual = userToStringConverter.convert(user);
+
+        assertNotNull(actual);
+        assertEquals(expected,actual);
```

Comment:
here you compare user with user
but in other tests you use getters. Why? Maybe we should make all tests looks the same